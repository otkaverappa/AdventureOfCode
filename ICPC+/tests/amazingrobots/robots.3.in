5 5
#####
#...#
#...#
#X#..
###.#
2
2 2 2 S
4 4 3 N
3 3
###
#X.
###
0
