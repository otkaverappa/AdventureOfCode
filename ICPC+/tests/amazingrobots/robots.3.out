9
E
E
E
N
E
E
S
E
E
